Update proposal.md

nikitagambhir opened the pull request:
@knausb @kaitlinmgold @sporangia This looks good! I updated the number of attendees to 50-65 based on our past experiences. And, I removed internet from the requirements because we typically ask attendees to download/install everything before coming to the workshop. Just in case somebody needs it, internet is always available. Another reason to remove this requirement is to make the workshop more affordable for attendees.

nikitagambhir committed:
Update proposal.md

@knausb @kaitlinmgold @sporangia This looks good! I updated the number of attendees to 50-65 based on our past experiences. And, I removed internet from the requirements because we typically ask attendees to download/install everything before coming to the workshop. Just in case somebody needs it, internet is always available. Another reason to remove this requirement is to make the workshop more affordable for attendees.

kaitlinmgold commented:
Hi all,

Proposal is due this week: are we all in agreement that the document is in
good shape to submit? Brian, do you want to take the lead on doing so? If
not I can do it.

Katie

*Kaitlin M. Gold *(Morey), *M.S., Ph.D.*
Asst. Professor of Grape Disease Ecology & Epidemiology, Cornell University
Faculty Fellow, Jet Propulsion Lab

Cell: 973-634-4215




On Wed, Aug 21, 2019 at 4:59 PM Nikita Gambhir <notifications@github.com>
wrote:

> @knausb <https://github.com/knausb> @kaitlinmgold
> <https://github.com/kaitlinmgold> @sporangia
> <https://github.com/sporangia> This looks good! I updated the number of
> attendees to 50-65 based on our past experiences. And, I removed internet
> from the requirements because we typically ask attendees to
> download/install everything before coming to the workshop. Just in case
> somebody needs it, internet is always available. Another reason to remove
> this requirement is to make the workshop more affordable for attendees.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/knausb/APS_IntroR_2020/pull/2
> Commit Summary
>
>    - Update proposal.md
>
> File Changes
>
>    - *M* proposal.md
>    <https://github.com/knausb/APS_IntroR_2020/pull/2/files#diff-0> (5)
>
> Patch Links:
>
>    - https://github.com/knausb/APS_IntroR_2020/pull/2.patch
>    - https://github.com/knausb/APS_IntroR_2020/pull/2.diff
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/knausb/APS_IntroR_2020/pull/2?email_source=notifications&email_token=AKO4GH5WMVMUZ6BTWK6GWUTQFXJEVA5CNFSM4IOPMP6KYY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4HGUXQNA>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKO4GH5PWQH33XPWWK4OYQTQFXJEVANCNFSM4IOPMP6A>
> .
>

nikitagambhir commented:
Hi everyone,

The proposal looks good to me. Either one of you may proceed for the
submission.

Thanks,
Nikita

On Mon, Aug 26, 2019 at 10:25 AM Katie Gold <notifications@github.com>
wrote:

> Hi all,
>
> Proposal is due this week: are we all in agreement that the document is in
> good shape to submit? Brian, do you want to take the lead on doing so? If
> not I can do it.
>
> Katie
>
> *Kaitlin M. Gold *(Morey), *M.S., Ph.D.*
> Asst. Professor of Grape Disease Ecology & Epidemiology, Cornell University
> Faculty Fellow, Jet Propulsion Lab
>
> Cell: 973-634-4215
>
>
>
>
> On Wed, Aug 21, 2019 at 4:59 PM Nikita Gambhir <notifications@github.com>
> wrote:
>
> > @knausb <https://github.com/knausb> @kaitlinmgold
> > <https://github.com/kaitlinmgold> @sporangia
> > <https://github.com/sporangia> This looks good! I updated the number of
> > attendees to 50-65 based on our past experiences. And, I removed internet
> > from the requirements because we typically ask attendees to
> > download/install everything before coming to the workshop. Just in case
> > somebody needs it, internet is always available. Another reason to remove
> > this requirement is to make the workshop more affordable for attendees.
> > ------------------------------
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/knausb/APS_IntroR_2020/pull/2
> > Commit Summary
> >
> > - Update proposal.md
> >
> > File Changes
> >
> > - *M* proposal.md
> > <https://github.com/knausb/APS_IntroR_2020/pull/2/files#diff-0> (5)
> >
> > Patch Links:
> >
> > - https://github.com/knausb/APS_IntroR_2020/pull/2.patch
> > - https://github.com/knausb/APS_IntroR_2020/pull/2.diff
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/knausb/APS_IntroR_2020/pull/2?email_source=notifications&email_token=AKO4GH5WMVMUZ6BTWK6GWUTQFXJEVA5CNFSM4IOPMP6KYY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4HGUXQNA
> >,
> > or mute the thread
> > <
> https://github.com/notifications/unsubscribe-auth/AKO4GH5PWQH33XPWWK4OYQTQFXJEVANCNFSM4IOPMP6A
> >
> > .
> >
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/knausb/APS_IntroR_2020/pull/2?email_source=notifications&email_token=AGDISL4QY5JAJJZHRQPRXIDQGPYXLA5CNFSM4IOPMP6KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD5EWY2Y#issuecomment-524905579>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGDISLY4PUGPNRILGO5XYWTQGPYXLANCNFSM4IOPMP6A>
> .
>


-- 
Nikita Gambhir
Graduate Research Assistant
Department of Plant Pathology
405 Plant Science Hall
University of Nebraska-Lincoln, NE  68583-0722
Telephone: (402)-975-9903
Email: nikitagambhir03@gmail.com

knausb commented:
Sounds like we're in agreement. Unless anyone objects, I'll try to submit this Tuesday.

knausb commented:
I've initiated the proposal. We apparently only get to have 3 "workshop organizers", so I put down @sporangia as a "moderator." @kaitlinmgold your affiliation, according to the APS database, is Wisconsin. Would you like to update to the affiliation you expect to have next summer? Thanks!

sporangia commented:
@knausb you are welcome to leave my name off the proposal.  I will not be assisting with this workshop.  If Nikita is unable to participate, I will step in as you and @kaitlinmgold need, but I would prefer that others can gain credit for this effort, not me.  :)

kaitlinmgold commented:
Ugh APS has been driving me crazy I've tried to update my affiliation 3
times already... will try again this afternoon.


*Kaitlin M. Gold *(Morey), *M.S., Ph.D.*
Asst. Professor of Grape Disease Ecology & Epidemiology, Cornell University
Faculty Fellow, Jet Propulsion Lab

Cell: 973-634-4215




On Tue, Aug 27, 2019 at 12:57 PM Sydney Everhart <notifications@github.com>
wrote:

> @knausb <https://github.com/knausb> you are welcome to leave my name off
> the proposal. I will not be assisting with this workshop. If Nikita is
> unable to participate, I will step in as you and @kaitlinmgold
> <https://github.com/kaitlinmgold> need, but I would prefer that others
> can gain credit for this effort, not me. :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/knausb/APS_IntroR_2020/pull/2?email_source=notifications&email_token=AKO4GH2XIQBG4Q4LWQBST7LQGWBJ3A5CNFSM4IOPMP6KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD5I5XGI#issuecomment-525458329>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKO4GH4IMYWHN2JFA7BFOFDQGWBJ3ANCNFSM4IOPMP6A>
> .
>

knausb commented:
Ok, we'll take @sporangia  off the list of names. @kaitlinmgold sorry if I'm the bearer of bad news. I had to update my affiliation and it was actually pretty easy. If there's anything I can try to do to help, let me know!
Brian

kaitlinmgold commented:
I had to get on the phone twice to get them to change my last name when I
got married... hopefully this will be easier!

*Kaitlin M. Gold *(Morey), *M.S., Ph.D.*
Asst. Professor of Grape Disease Ecology & Epidemiology, Cornell University
Faculty Fellow, Jet Propulsion Lab

Cell: 973-634-4215




On Tue, Aug 27, 2019 at 1:48 PM Brian Knaus <notifications@github.com>
wrote:

> Ok, we'll take @sporangia <https://github.com/sporangia> off the list of
> names. @kaitlinmgold <https://github.com/kaitlinmgold> sorry if I'm the
> bearer of bad news. I had to update my affiliation and it was actually
> pretty easy. If there's anything I can try to do to help, let me know!
> Brian
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/knausb/APS_IntroR_2020/pull/2?email_source=notifications&email_token=AKO4GHZHYFMH253D2CW6MX3QGWHKRA5CNFSM4IOPMP6KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD5JCGYY#issuecomment-525476707>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKO4GH5VT4JAYWLZQH2XZMDQGWHKRANCNFSM4IOPMP6A>
> .
>

kaitlinmgold commented:
My affiliation should be changed now- or at least thats what the portal
says.


*Kaitlin M. Gold *(Morey), *M.S., Ph.D.*
Asst. Professor of Grape Disease Ecology & Epidemiology, Cornell University
Faculty Fellow, Jet Propulsion Lab

Cell: 973-634-4215




On Tue, Aug 27, 2019 at 1:56 PM Kaitlin M. Gold <kaitlin.m.gold@gmail.com>
wrote:

> I had to get on the phone twice to get them to change my last name when I
> got married... hopefully this will be easier!
>
> *Kaitlin M. Gold *(Morey), *M.S., Ph.D.*
> Asst. Professor of Grape Disease Ecology & Epidemiology, Cornell
> University
> Faculty Fellow, Jet Propulsion Lab
>
> Cell: 973-634-4215
>
>
>
>
> On Tue, Aug 27, 2019 at 1:48 PM Brian Knaus <notifications@github.com>
> wrote:
>
>> Ok, we'll take @sporangia <https://github.com/sporangia> off the list of
>> names. @kaitlinmgold <https://github.com/kaitlinmgold> sorry if I'm the
>> bearer of bad news. I had to update my affiliation and it was actually
>> pretty easy. If there's anything I can try to do to help, let me know!
>> Brian
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/knausb/APS_IntroR_2020/pull/2?email_source=notifications&email_token=AKO4GHZHYFMH253D2CW6MX3QGWHKRA5CNFSM4IOPMP6KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD5JCGYY#issuecomment-525476707>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AKO4GH5VT4JAYWLZQH2XZMDQGWHKRANCNFSM4IOPMP6A>
>> .
>>
>
